lib/App/Sqitch/Role/DBIEngine.pm

```diff
@@ -756,7 +756,7 @@ sub deployed_changes {
     } @{ $self->dbh->selectall_arrayref(qq{
         SELECT c.change_id AS id, c.change AS name, c.project, c.note,
                $tscol AS "timestamp", c.planner_name, c.planner_email,
-               $tagcol AS tags
+               $tagcol AS tags, c.script_hash
```

Comment:
The addition of `script_hash` makes a lot of sense here, but you almost certainly need to update unit tests for these methods, especially in `t/engine.t` and `t/lib/DBIEngineTest.pm`. Also check the specific engines (subclasses of App::Sqitch::Engine) to see if any override these methods, as those overrides will need to be modified, too.